src/libs/actions/App.js

```diff
@@ -135,6 +144,43 @@ function fixAccountAndReloadData() {
         });
 }
 
+/**
+ * Wait for the navigation to be ready, get the initial URL and make sure the
+ * user is logged in. Next, create a new free policy if needed, load policies,
+ * and navigate to the transition exit route if needed.
+ */
+function setUpPoliciesAndNavigate() {
+    Navigation.isNavigationReady()
+        .then(Linking.getInitialURL)
+        .then((url) => {
+            if (!lodashGet(session, 'authToken', null)) {
```

Comment:
Hmm no way is there no `authToken` right? This is called from "_Auth_ Screens" so how is it possible?